src/Riok.Mapperly/MapperGenerator.cs

```diff
@@ -72,6 +74,25 @@ CancellationToken cancellationToken
         var diagnostics = new List<Diagnostic>();
         var members = new List<MapperNode>();
 
+        var defaultMapperAttributeSymbol = wellKnownTypes.TryGet(DefaultMapperAttributeName);
+        var defaultMapperAttributes = compilation.Assembly
+            .GetAttributes()
+            .Where(x => SymbolEqualityComparer.Default.Equals(x.AttributeClass, defaultMapperAttributeSymbol))
+            .ToList();
+
+        if (defaultMapperAttributes.Count > 1)
+        {
+            diagnostics.Add(
+                Diagnostic.Create(DiagnosticDescriptors.CannotUseMultipleAssemblyAttributes, null, typeof(DefaultMapperAttribute))
+            );
+        }
+
+        var defaultMapperAttribute = defaultMapperAttributes.FirstOrDefault();
+        var defaultConfiguration =
+            defaultMapperAttribute != null
+                ? AttributeDataAccessor.Access<DefaultMapperAttribute, MapperConfiguration>(defaultMapperAttribute)
+                : null;
```

Comment:
To reduce this method's complexity, I'd extract this into a separate method.
Couln't this be extracted in another incremental step? I think this could improve the overall performance.